nilearn/datasets/tests/test_utils.py

```diff
@@ -364,7 +364,7 @@ def test_uncompress_tar(tmp_path, ext, mode):
     # and check if ftemp exists
     ztemp = tmp_path / f"test.{ext}"
     ftemp = "test"
-    with contextlib.closing(tarfile.open(ztemp, mode)) as testtar:
+    with tarfile.open(ztemp, mode) as testtar:
```

Comment:
Pretty sure the `contextlib.closing` is not needed : https://docs.python.org/3/library/tarfile.html#tarfile-objects